src/libraries/System.Text.Json/src/System/Text/Json/Serialization/ConfigurationList.cs

```diff
@@ -116,5 +116,22 @@ IEnumerator IEnumerable.GetEnumerator()
         {
             return _list.GetEnumerator();
         }
+
+        internal string DebuggerDisplay
+        {
+            get
+            {
+                var sb = new StringBuilder("[");
+                foreach (TItem item in _list)
+                {
+                    sb.Append(item);
+                    sb.Append(", ");
+                }
+
+                sb.Length -= 2;
```

Comment:
>sb.Length -= 2; [](http://example.com/codeflow?start=16&length=15)

Is it possible ever we can have empty list in any situation?

Reply:
Good catch, I should have guarded that with check.